StyleCop.Analyzers/StyleCop.Analyzers/DocumentationRules/SA1648InheritDocMustBeUsedWithInheritingClass.cs

```diff
@@ -86,7 +86,7 @@ private static void HandleBaseTypeLikeDeclaration(SyntaxNodeAnalysisContext cont
 
             if (documentation.Content.GetFirstXmlElement(XmlCommentHelper.IncludeXmlTag) is XmlEmptyElementSyntax includeElement)
             {
-                var declaration = context.SemanticModel.GetDeclaredSymbol(baseType, context.CancellationToken);
+                var declaration = baseType is null ? context.ContainingSymbol : context.SemanticModel.GetDeclaredSymbol(baseType, context.CancellationToken);
```

Comment:
💭 It seems like the intent would be closer to this:

```suggestion
                var declaration = context.SemanticModel.GetDeclaredSymbol(context.Node, context.CancellationToken);
```